### PR TITLE
Tests for DSL

### PR DESCRIPTION
**Pull request by LTRocky**

### CODE REVIEW

File: DSL/Query/QueryAwareTrait.php [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            $this->queries[$query->getType()] = $query;
+        }
+
+        return $this;
+    }
+
+    /**
+     * Returns Bool query. Creates new instance if there is not initiated.
+     *
+     * @return \ONGR\ElasticsearchBundle\DSL\Bool\Bool
```
Comment (lmikelionis) [inline DSL/Query/QueryAwareTrait.php]: Its better to use short version

**Comment by martiis** [comment]
Please rebase all merge commits

File: Tests/Unit/DSL/ParametersTraitTest.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        $this->mock = $this->getMockForTrait('ONGR\ElasticsearchBundle\DSL\ParametersTrait');
+    }
+
+    /**
+     * Tests getParameter method.
+     */
+    public function testGetParameter()
+    {
+        $this->assertFalse($this->mock->getParameter('unavailable_parameter'));
+
```
Comment (lmikelionis) [inline Tests/Unit/DSL/ParametersTraitTest.php]: I think we can do without /n

File: Tests/Unit/DSL/Query/FilteredQueryTest.php [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        $this->assertEquals($expected, $result);
+    }
+
+    /**
+     * Tests getType method.
+     */
+    public function testGetType()
+    {
+        $query = new FilteredQuery();
+
```
Comment (lmikelionis) [inline Tests/Unit/DSL/Query/FilteredQueryTest.php]: no need for /n here

File: Tests/Unit/DSL/Query/HasChildQueryTest.php [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+{
+    /**
+     * Tests whether __constructor calls setParameters method.
+     */
+    public function testConstructor()
+    {
+        $missingFilter = $this->getMockBuilder('ONGR\ElasticsearchBundle\DSL\Filter\MissingFilter')
+            ->setConstructorArgs(['test_field'])
+            ->getMock();
+
```
Comment (lmikelionis) [inline Tests/Unit/DSL/Query/HasChildQueryTest.php]: I think no need for /n here

File: Tests/Unit/DSL/Query/HasParentQueryTest.php [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+{
+    /**
+     * Tests whether __constructor calls setParameters method.
+     */
+    public function testConstructor()
+    {
+        $missingFilter = $this->getMockBuilder('ONGR\ElasticsearchBundle\DSL\Filter\MissingFilter')
+            ->setConstructorArgs(['test_field'])
+            ->getMock();
+
```
Comment (lmikelionis) [inline Tests/Unit/DSL/Query/HasParentQueryTest.php]: I think, no need for /n here

File: Tests/Unit/DSL/Query/NestedQueryTest.php [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+        $result = [
+            'path' => 'test_path',
+            'query' => [
+                'test_type' => ['testKey' => 'testValue'],
+            ],
+        ];
+
+        $query = new NestedQuery('test_path', $missingFilter);
+
```
Comment (lmikelionis) [inline Tests/Unit/DSL/Query/NestedQueryTest.php]: I think, no need for /n here

File: Tests/Unit/DSL/Query/QueryTest.php [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        $missingFilter = $this->getMockBuilder('ONGR\ElasticsearchBundle\DSL\Filter\MissingFilter')
+            ->setConstructorArgs(['test_field'])
+            ->getMock();
+
+        $missingFilter->expects($this->once())
+            ->method('setParameters');
+
+        $query = new Query();
+
```
Comment (lmikelionis) [inline Tests/Unit/DSL/Query/QueryTest.php]: definetly, no need for /n lines here (block below)

File: Tests/Unit/DSL/SearchTest.php [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+     */
+    public function testAddAndDestroyQuery()
+    {
+        $search = new Search();
+        $queryMock = $this->getMock('ONGR\ElasticsearchBundle\DSL\Query\Query');
+
+        $this->assertEmpty($search->getQueries());
+        $search->addQuery($queryMock);
+        $this->assertNotNull($search->getQueries());
+
```
Comment (lmikelionis) [inline Tests/Unit/DSL/SearchTest.php]: maybe no need for /n here? :)

**Comment by saimaz** [comment]
Please do it properly!

**Pull request closed without merging**